packages/strapi-generate-new/files/config/environments/development/request.json

```diff
@@ -10,13 +10,14 @@
     }
   },
   "logger": {
-    "enabled": true
+    "enabled": false,
```

Comment:
The logger can't be disabled. So, this configuration should be called `exposeInContext`.